yesod-core/test/YesodCoreTest/Exceptions.hs

```diff
@@ -41,7 +41,7 @@ case500 :: IO ()
 case500 = runner $ do
     res <- request defaultRequest
     assertStatus 500 res
-    assertBody "FOOBAR" res
+    assertBodyContains "FOOBAR" res
```

Comment:
Hmm, i really should not respond to github issues before my morning coffee :).

Without the change from `assertBody` to `assertBodyContains` after building with ghc-8.0, the test fail with:

```
  1) Test.Exceptions 500
       uncaught exception: WaiTestFailure (WaiTestFailure "Expected response body \"FOOBAR\", but received \"FOOBAR\\nCallStack (from HasCallStack):\\n  error, called at test/YesodCoreTest/Exceptions.hs:25:12 in main:YesodCoreTest.Exceptions\"")
```

which points directly at line 25 which is:

```
getRootR = error "FOOBAR" >> return ()
```

So, `error` is relevant, because `error` now has a `CallStack`.